public/locales/pt/createPet.json

```diff
@@ -0,0 +1,85 @@
+{
+  "createPetTitle": "Crie um novo mascote",
+  "editTo": "Alterar {{ name }}",
+  "backPets": "Volta do perfil",
+  "placeholderImages": "Adicionar imagem do mascote",
+  "addFile": "Escolher uma imagen",
+  "assignUserTransit": "Atribuir usuário de trânsito",
+  "assignUser": "Atribuir adotante",
+  "labelWhereFoundPet": "¿Onde foi encontrada?",
+  "labelSelectTransit": "Atribuir usuário de trânsito",
+  "placeHolderSelectTransit": "Escolher um usuário...",
+  "placeholderName": "Nome da mascote",
+  "adopted": "¿O mascote foi adotada?",
+  "statusPet": "Estado da mascote",
+  "labelWhoAdopted": "¿Quem adotou los {{ name }}?",
```

Comment:
hmm...
here we have a problem cause in portuguese we have a problem the genre flexion.
I suggest to use: "Quem adotou o/a {{ name }}?"